packages/webapp/pages/devcard.tsx

```diff
@@ -345,7 +345,7 @@ const DevCardPage = (): ReactElement => {
   return (
     <div
       className={classNames(
-        'page flex min-h-screen flex-col items-center px-6 py-10 laptop:-mt-12 max-w-full laptop:justify-center mx-auto',
+        'page flex min-h-page flex-col justify-center items-center px-6 py-10 tablet:-mt-12 max-w-full mx-auto',
```

Comment:
Did you double check other pages that might have used `min-h-screen` before?
Just concerned some global change caused this in the first place.

Reply:
I checked the other pages and seem to be working fine. We didn't remove the `min-h-screen` but I'm unsure what actually caused it. Will further investigate to avoid future occurences.